Josue/index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Exercise</title>
+    <!--ICONS-->
+    <script
+      defer
+      src="https://use.fontawesome.com/releases/v5.0.7/js/all.js"
+    ></script>
+    <!--LATO FONT-->
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="container navbar">
+        <ul class="navbar__list">
+          <li class="navbar__list-logo">
+            <a href="/"><img src="" alt="Logo" /></a>
+          </li>
+          <li class="navbar__list-items">
+            <a href="#">Features</a>
+            <a href="#">Templates</a>
+            <a href="#">Integrations</a>
+            <a href="#">Pricing</a>
+          </li>
+          <li class="navbar__list-buttons">
+            <button class="btn btn--filled">Sign In</button>
+            <button class="btn btn--outline">Sign Up</button>
+          </li>
+        </ul>
+      </nav>
+
+      <section class="hero">
+        <div class="hero__content">
+          <h1>Hola soy Josué</h1>
+          <h2>Desarrollador Web</h2>
+        </div>
+      </section>
+    </header>
+
+    <section class="container achievements">
+      <!--Using ul-->
+      <ul class="achievements__list">
+        <li>
+          <i class="far fa-image fa-2x"></i>
+          <p>Años de experiencia</p>
+        </li>
+        <li>
+          <i class="far fa-image fa-2x"></i>
+          <p>Galardones</p>
+        </li>
+        <li>
+          <i class="far fa-image fa-2x"></i>
+          <p>Clientes Satisfechos</p>
+        </li>
+        <li>
+          <i class="far fa-image fa-2x"></i>
+          <p>Proyectos Concluidos</p>
+        </li>
+      </ul>
+    </section>
+    <hr />
+
+    <main class="container main-content">
+      <!--PARTE UNO-->
+      <section class="main-content__left">
+        <!--Un heading-->
+        <p>THE AMAZING HUBBLE</p>
+        <h2>Spend Less Time Coding</h2>
+        <div class="main-content__left-post">
+          <figure>
+            <img
+              src="https://picsum.photos/seed/picsum/200/300"
+              alt="Random Image"
+            />
+            <figcaption>Just another random image</figcaption>
+          </figure>
+          <div class="main-content__blog">
```

Comment:
Change div for an article

Reply:
Should I use article instead of div? The div was created to wrap the two articles "Become a power-user" and "Know how to use Vim" and place them next to the picture. Is it correct? or what should I do?